config/crd/bases/kustomize.toolkit.fluxcd.io_kustomizations.yaml

```diff
@@ -305,10 +323,6 @@ spec:
                   When not specified, the controller uses the KustomizationSpec.Interval
                   value to retry failures.
                 type: string
-              serviceAccountName:
```

Comment:
We can't remove this filed from the `v1beta1` API. If `principal` is `nil` and if `serviceAccountName` is specified, the controller should behave like `principal` is set to that service account name.